src/rest/subhandlers/rest_events.erl

```diff
@@ -0,0 +1,13 @@
+%%%-------------------------------------------------------------------
+%%% @author pawel
+%%% @copyright (C) 2021, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 25. May 2021 20:43
+%%%-------------------------------------------------------------------
+-module(rest_events).
+-author("pawel").
+
+%% API
+-export([]).
```

Comment:
Rozumiem że ten plik i inne w tym stylu to placeholder? Jak tak, to w sumie dodaj jakiś komentarz do tego, bo aktualnie to jest lekkie wtf

Reply:
Ok, skoro sobie życzysz. Są to moduły do tych ścieżek/funkcjonalności API, które będą implementowane później, ale już je zrobiłem, żeby je mieć pod ręką i zobaczyć ile ich w ogóle będzie